src/common/Select/Select.tsx

```diff
@@ -77,7 +84,11 @@ export const Select: SelectComponent<SelectProps> = ({
   );
   const [focused, setFocused] = useState<string | null>();
 
-  const options = Children.toArray(children);
+  const options = Children.toArray(children).filter(child => {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    return child.type.displayName === "Select.Option" && !child.props.exclude;
```

Comment:
if possible lets figure out and resolve the issue instead of using ts-ignore. if I'm reading it correctly you are looking to filter the child elements and find only options? at a glance I dont see what would trigger a TS error here

Reply:
Turns out the type is wrong, it is even re-cast elsewhere in that file to any. I removed the ts-ignore and opted to do the same treatment, adding a comment about why.